content/case_histories/hoop_region_norway/processinginterpretation.rst

```diff
@@ -0,0 +1,207 @@
+
+
+.. figure:: images/inversion_initial_pwave_poisson.png
+    :align: center
+    :figwidth: 100%
+    :name: fig_inversion_initial_pwave_poisson
+
+    Initial model, P-wave impedance and Poisson ratio from seismic inversion (left), tracks of P-wave impedance, Poisson ratio, volume of clay and water saturation extracted along the projection of the well 7324/8-1 (right).
+
+
+The multi-attribute rotation scheme (MARS) (Alvarez et al. 2015), was used to estimate rock properties and facies volumes from well log and seismic inversion attributes. This workflow uses a numerical solution to estimate a transform to predict petrophysical properties from elastic attributes. The transform is computed from well log-derived elastic attributes and petrophysical properties, and posteriorly applied to seismically-derived elastic attributes. MARS estimates a new attribute, t, in the direction of maximum change of a target property in an n-dimensional Euclidean space formed by n attributes. The method sequentially searches for the maximum correlation between the target property and all of the possible attributes that can be estimated via an axis rotation of the basis that forms the aforementioned space.
+
+Multiple elastic attributes such as P-wave impedance IP, S-wave impedance Is, P-to-S velocity ratio (VP ∕ Vs), the product of density and Lamé’s parameters λρ and μρ (Goodway et al., 1997), Poisson’s ratio σ, the product of density by bulk modulus Kρ, the product of density and dynamic Young’s modulus Eρ, Poisson dampening factor (PDF) (:cite:`Mazumdar2007`), etc., can be used in the MARS assessment. For this case study, for each target petrophysical property, MARS was run for a 2D combination of the 64 elastic attributes shown in :numref:`fig_table_seismic_attributes` , which can be derived from IP and IS, resulting in the evaluation of 2016 independent bi-dimensional spaces. In this table, each number represents a single attribute, which is obtained after applying the mathematical operation shown in the leftmost column to the elastic attribute shown in the uppermost row. For example, the number 21 represents the attribute. The purpose of applying a mathematical operation (such as square root, power, inverse, logarithm, etc.) to attributes is to be able to model physical phenomena that exhibit nonlinear behavior. This is a mathematical strategy to linearize potential nonlinear relationship between the elastic attributes and the petrophysical properties, used with the goal of improving the correlation between the attribute t and the target petrophysical property.
```

Comment:
Probably. As it stands, I wouldn't be surprised if paragraph like these got re-written. So I didn't bother making changes if it isn't the final text